### PR TITLE
Added CDNI Signed Token Depth claim.

### PR DESCRIPTION
**Pull request by alficles**
This claim will allow CDNs to determine what path should be used
for cookies and other Signed Token Renewal Transports.

Also, I apparently changed a bunch of tabs to spaces in my editor.

### CODE REVIEW

**Comment by PSUdaemon** [comment]
I think this is good. Can you add it to the example for signed token renewal?

Also, can you fix the whitespace changes?

**Comment by alficles** [comment]
I _can_ fix the whitespace changes. :) But those are the only spots where tabs are used at all. This makes it possible to `s/^I/        /` to get rid of spurious tabs.

And yeah, I'll add it to the example.

File: draft-ietf-cdni-uri-signing.xml [hunk 1]
```diff
@@ -1743,9 +1763,10 @@ SWi9agE9vtuDsOzkbBGgd691XtjrjL-JO_g
         </t>
 
         <figure><artwork><![CDATA[
-{
+
```
Comment (PSUdaemon) [inline draft-ietf-cdni-uri-signing.xml]: Is removing this intended?

Comment (alficles) [reply draft-ietf-cdni-uri-signing.xml]: No.

**Pull request merged**